developerSandboxSetup.md

```diff
@@ -31,100 +31,174 @@ There are three options for running a sandbox.
-* Fctwallet is an application for holding private keys. It builds Factoid transactions and handles crypto related operations to add user data into Factom.
-* Factom-cli is a program for users to interface with factomd and fctwallet. Through this they create Chains, Entries, and Factoid transactions.
+* factom-walletd is an application for holding private keys. It builds Factoid transactions and handles crypto related operations to add user data into Factom.
+* Factom-cli is a program for users to interface with factomd and factom-walletd. It may be used to create Chains, Entries, and Factoid transactions.
 
 #### Configure Factomd for Sandbox use
 
+Create a folder in your user home folder. The folder should be called `.factom`. If factomd has been run on your computer before this folder will already exist and may already contain part of the factom blockchain.
 
-Create a folder in your user home folder. The folder should be called `.factom`. If factomd has been run on your computer before you may need to first rename the database that is already there: `mv ~/.factom/ldb ~/.factom/ldb_old`.
+In a terminal Linux and Mac: `mkdir -p ~/.factom/m2`or Windows: `mkdir %HOMEDRIVE%%HOMEPATH%\.factom\m2`
 
-In a terminal Linux and Mac: `mkdir ~/.factom`or Windows: `mkdir %HOMEDRIVE%%HOMEPATH%\.factom`
-
-Save the configuration file [factomd.conf](https://raw.githubusercontent.com/FactomProject/FactomCode/master/factomd/factomd.conf) to your `.factom` directory.
+Save the configuration file [factomd.conf](https://raw.githubusercontent.com/FactomProject/factomd/master/factomd.conf) to your `.factom/m2` directory.
 
 * Open `factomd.conf`
+* Change the line `Network` from `MAIN` to `LOCAL`
+	* This will cause factomd to run on its own local network rather than the factom main network.
 * Change the line `NodeMode` from `FULL` to `SERVER`
   * This will make factomd create a blockchain
-* Change `ExchangeRate` to `00000100`
-  * This allows you to create many more Entries with Factoids.  It simulates Factoids having a high market value.
 * Optionally adjust `DirectoryBlockInSeconds`.  600 gives 10 minute blocks, which is more realistic
-  * The default is for 1 minute blocks (60) which is much easier to develop with.  Shorter times have not been tested.
+  * The default is for 6 second blocks which is easier to develop with, but may cuase some strange behavior and errors.
 
 For easier debugging, change the `logLevel` to  `debug`.  This exports pointers to data which is added into factom to the ~/.factom/data/export/ directory.  Adding new entries will add new files to the directory with the specified ChainID.
 
+#### Run Factomd in sandbox mode
 
-#### Run Factomd
-
-In a terminal window, run `factomd`. (Windows users have a desktop shortcut)
+In a terminal window, run `factomd -network=CUSTOM -customnet="mycustomnet" -exclusive=true`. (Windows users have a desktop shortcut)
```

Comment:
we got rid of the windows shortcut because for nic the directions would be easier and it would be easier to debug local problems if the terminal window doesn't automatically close.